modules/appeals_api/app/controllers/concerns/appeals_api/openid_auth.rb

```diff
@@ -23,7 +23,7 @@ module OpenidAuth
     }.freeze
 
     included do
-      before_action :validate_auth_token!
+      prepend_before_action :validate_auth_token!
```

Comment:
I also included this one-line change to make sure oauth tokens are validated before request headers are